wp-content/themes/core/blocks/core/button/style.pcss

```diff
@@ -1,6 +1,104 @@
 /**
  * Styles specific to this block
  */
-.wp-block-button {
-	outline: 1px solid #f00;
+
+.wp-block-button__link {
+	transition: var(--transition);
+}
+
+/* -------------------------------------------------------------------------
+ * Button: Arrow After Label
+ * ------------------------------------------------------------------------- */
+
+:is(.is-style-primary, .is-style-secondary) .wp-block-button__link {
```

Comment:
Any chance we can set up the the styles as parent-selectors (nested) throughout this file? Something like:

```
.wp-block-button__link {
	
		:is(.is-style-primary, .is-style-secondary) & { 
			...
		}
```

Reply:
Yes, will do 👍